benches/oneshot/ipa.rs

```diff
@@ -28,21 +27,20 @@ async fn main() -> Result<(), Error> {
     let mut total_count = 0;
 
     let mut random_user_records = Vec::with_capacity(NUM_USERS);
```

Comment:
This is your only use for NUM_USERS now.

I would suggest that you calculate this value.  Setting it to QUERY_SIZE / Expected(records_per_user) is probably better than keeping a constant that will rot.

Also, I think we should change the distribution from which we draw the number of records for each user.  Uniform is not that realistic.

Reply:
we actually don't draw from a fixed number of users. Inside the function, it generates a random number upto MAX_RECORDS_PER_USER and uses that to decide the count.